symbolic-minidump/src/cfi.rs

```diff
@@ -41,9 +42,23 @@ use symbolic_debuginfo::pdb::PdbObject;
 use symbolic_debuginfo::pe::{PeObject, RuntimeFunction, UnwindOperation};
 use symbolic_debuginfo::{Object, ObjectError, ObjectLike};
 
+/// The magic file preamble to identify cficache files.
+/// Files with version < 2 do not have a magic file preamble.
```

Comment:
They don't have the `CFIC` preamble, but you validate version 1 files by checking that they start with a breakpad `STACK ` record.

Separately, can we declare that this preamble is subject to endianness and document how it behaves? Especially, that we read and write with native endianness.